### DISCUSSION TITLE
#43 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Riely007** [tog-ETH-er/togETHer#20] [General]
Agenda:

1. 下一届轮值主席提名讨论
2. Community updates
3. Fireside discussion

> 会议主题：togETHer
> 会议时间： 21:00-22:00
> 重复周期：每两周 (周一)
> 
> 点击链接入会，或添加至会议列表：
> https://meeting.tencent.com/s/LxNajKWJw4RV
> 
> 会议 ID：891 5098 6775

**Comment by 0x-stan** [comment]
# Dapp-Learning

## repo update

10个PR #793 ～ #802

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了4场分享会

- 《DA技术简析》by cyberorange
  - https://www.youtube.com/watch?v=JL8CzYgqcPk
  - https://www.youtube.com/watch?v=_-THtzv5vEQ 

- 《Black-Scholes模型从入门到放弃》by Kaiji

- 《从金融角度理解推导BS公式》by Jezex
  - https://www.youtube.com/watch?v=KLSG1W-oqhI
  - https://www.youtube.com/watch?v=dq1nyjs5VrI

- 《arbitrum中的gas机制》by JasonWan
  - https://www.youtube.com/watch?v=EvnuU6PSxEU&t=9s

## 翻译小组

- MIT公开课准备发布第20集

**Reply by bambalados** [reply]
近两周举办了2场线上活动:

- CTN#15 "[区块链与大学](https://t.co/nEe8pmreBa)" (腾讯会议)
- CTN Let's Talk2 "[The Merge要延期？Core Dev Meeting到底聊了什么 + 对我们有什么影响](https://twitter.com/i/spaces/1mrGmaaDeOzGy)" (Twitter Space)

**Comment by stephanieeee-ho** [comment]
### ECN
- 日常翻译和七日谈
- 参加了 CTN "The Merge要延期？”的 twitter space

**Comment by winkybupt** [comment]
## 原语里弄

1. 难度炸弹简史 报告发布

**Comment by bobjiang** [comment]
1. GitcoinDAO cn newsletter - http://news.bobjiang.com/
2. Beijing conf (Nov.)
3. ENS ecosystem, any idea?
4.

**Comment by liushooter** [comment]
## Rebase

### 安排

- infura 工程师介绍 devops for blockchian

### 活动

- 在杭州举办了一场 rebase meetup

- 举办了一期 Codingday 活动
> ![](https://user-images.githubusercontent.com/1235106/178268745-5044363d-b26f-4875-b2af-afbf05ba1db9.png)

- [Web3 日报](https://rebase.network/posts/category/web3daily) 正常更新 14 期

- 更新多篇 [招聘](https://github.com/rebase-network/who-is-hiring)

- 更新多篇 [活动](https://github.com/rebase-network/event-pool)

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 官网翻译进行中

**Comment by 0x68616f79756e** [comment]
**ETHPlanet**
筹备 EDCON 
筹备 以太坊之夏

**Comment by Emilyxiannv** [comment]
### DAOSquare
内容日常更新，无特别分享，更多参考 dao2.muse.lol 以及daosquare.muse.lol
